src/Draggable/tests/Draggable.test.js

```diff
@@ -418,16 +418,21 @@ describe('Draggable', () => {
       .toBeInstanceOf(DragStartEvent);
   });
 
-  test('sets dragging to false when `drag:start` event is canceled', () => {
+  test('cleans up when `drag:start` event is canceled', () => {
     const newInstance = new Draggable(containers, {
       draggable: 'li',
     });
     const draggableElement = sandbox.querySelector('li');
     document.elementFromPoint = () => draggableElement;
 
+    let source;
+    let originalSource;
     const callback = jest.fn((event) => {
+      source = event.data.source;
+      originalSource = event.data.originalSource;
```

Comment:
You can use `event.source` and `event.originalSource`. Which reminds me, we should make `data` a private member